snapcraft/tests/test_plugin_rust.py

```diff
@@ -86,7 +86,7 @@ def test_pull(self, run_mock, script_mock):
 
         plugin.pull()
 
-        self.assertEqual(1, run_mock.call_count)
+        self.assertEqual(2, run_mock.call_count)
```

Comment:
This test would require more than checking that rust was called twice. Please take a look at the next statement that checks the arguments of the first call, and do something similar to the second.